packages/core-web/src/styles/markbind.css

```diff
@@ -333,3 +333,14 @@ h5,
 h6 {
     margin-bottom: 1rem;
 }
+
+.block-embed {
+    position: relative;
+    padding-bottom: 56.25%; /* 16:9 */
+}
```

Comment:
We can also add the `overflow: hidden` and `height: 0` for robustness like the sample fiddle in the issue (reasons why it's needed are outlined [here](https://www.benmarshall.me/responsive-iframes/) at Things to Remember)

Reply:
> The particular `56.25%` value suits well with the YouTube embed as the set dimension is `640x390`, which is `16:9` (see that `100 / (16 / 9) = 56.25`). However, PowerPoint embeds set its dimension to `610x481`, which requires a different percentage, but can be easily calculated by hand like above and put the approximate into the styling.

`640x390` is not 16:9 though? The percentage calculations ended up being just a rough estimate on how to reduce the blank space in the view area. 

e.g. 
![image](https://user-images.githubusercontent.com/19247856/156631933-cf5c8946-9249-48b1-a7f2-f8e9beab05e5.png)
vs 
![image](https://user-images.githubusercontent.com/19247856/156631970-4d16523a-c061-482f-a6d6-d8eae1f38954.png)

I'm also not sure how the original author decided on each width/ height value also, but would it be ok to make changes to them?

Reply:
yep i've made the changes to scale based on the width/height of the container